src/js/common/schemaform/definitions/address.js

```diff
@@ -120,12 +124,22 @@ export function uiSchema(label = 'Address', useStreet3 = false) {
       validateAddress
     ],
     'ui:options': {
-      updateSchema: (formData, addressSchema, addressUiSchema, index, path) =>
-        addressChangeSelector({
+      updateSchema: (formData, addressSchema, addressUiSchema, index, path) => {
+        let currentSchema = addressSchema;
+        if (isRequired) {
```

Comment:
I don't want to call `_.set()` when the output of `isRequired()` hasn't changed, as that means we'll always create a new copy of addressSchema, which will cause the addressSelector to run more often than it needs to and force the address on the page to re-render unnecessarily.